### DISCUSSION TITLE
Welcome to NukeSurvivalToolkit Discussions!

### DISCUSSION THREAD

**Discussion by CreativeLyons** [CreativeLyons/NukeSurvivalToolkit_publicRelease#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by manishmishraguru999** [comment]
its not working in nuke 13

**Comment by maheshvisual** [comment]
Hi all,

I have configured NukeSurvivalToolkit as per the demo video, it's launching well with nukex12.0v1
But it's not launching with nuke 15.1v3.
Error is :  
File "C:/Users/MAHESH/.nuke/init.py", line 5
    nuke.pluginAddPath("G:\Sys Soft\VFX Soft\Nuke\Plugins\NukeSurvivalToolkit_publicRelease-2.1.1\NukeSurvivalToolkit")
                                                                                                                      ^
SyntaxError: (unicode error) 'unicodeescape' codec can't decode bytes in position 20-21: malformed \N character escape

Previously I got the same issue with Nuke 13 as well(due to this i stopped using 13)
![erro6](https://github.com/user-attachments/assets/79739bc4-25e0-4ef4-9f60-ca5bf8044881)

**Reply by brettmaker** [reply]
Hi Manesh,

You'll need to use double backslashes because a single backslash as an escape character.
```
nuke.pluginAddPath("G:\\Sys Soft\\VFX Soft\\Nuke\\Plugins\\NukeSurvivalToolkit_publicRelease-2.1.1\\NukeSurvivalToolkit")
```
Forward slashes will also work in the newer versions of Windows:
```
nuke.pluginAddPath("G:/Sys Soft/VFX Soft/Nuke/Plugins/NukeSurvivalToolkit_publicRelease-2.1.1/NukeSurvivalToolkit")
```

**Reply by maheshvisual** [reply]
Hi brettmaker,

Thanks, it's working fine.
